### PR TITLE
Nebula kotlin nodep

### PR DESCRIPTION
**Pull request by rpalcolea**
`org.jetbrains.kotlin:kotlin-stdlib` not applied if `nodep` is used

### CODE REVIEW

File: src/test/groovy/netflix/nebula/NebulaKotlinNodepPluginIntegrationSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        buildFile << """
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        """
+
+        when:
+        def resultCompileClasspath = runTasksSuccessfully('dependencies', '--configuration', 'compileClasspath')
+
+        then:
+        !resultCompileClasspath.standardOutput.contains("\\--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlinVersion\n")
+    }
```
Comment (OdysseusLives) [inline src/test/groovy/netflix/nebula/NebulaKotlinNodepPluginIntegrationSpec.groovy:72]: Can you add a test case in here where ppl apply the no-dep plugin and add dependencies to a configuration explicitly?

Comment (rpalcolea) [reply src/test/groovy/netflix/nebula/NebulaKotlinNodepPluginIntegrationSpec.groovy:72]: added here https://github.com/nebula-plugins/nebula-kotlin-plugin/pull/34/commits/4dc0c5f44bf91751f1c33a462bc05632c60fbce4

Comment (OdysseusLives) [reply src/test/groovy/netflix/nebula/NebulaKotlinNodepPluginIntegrationSpec.groovy:72]: 🎉

**Review by OdysseusLives — APPROVED** [review]
:shipit:

**Pull request merged**